po/noVNC.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: noVNC 0.6.1\n"
 "Report-Msgid-Bugs-To: novnc@googlegroups.com\n"
-"POT-Creation-Date: 2016-11-15 19:32+0100\n"
+"POT-Creation-Date: 2016-11-24 08:53+0200\n"
```

Comment:
As you can see you didn't need to update the `.pot` file for this. Please remove this from the commit to avoid polluting the history.